src/libratbag-data.c

```diff
@@ -104,6 +106,13 @@ init_data_hidpp10(struct ratbag *ratbag,
 	if (error)
 		g_error_free(error);
 
+	error = NULL;
+	num = g_key_file_get_integer(keyfile, group, "LEDs", &error);
```

Comment:
Change it to `Leds` please, we're using `Dpi` instead of `DPI` in those files too. And add the entry to the `device.example` file so it's documented.